resources/js/upload.js

```diff
@@ -0,0 +1,41 @@
+var fileTypes = ['pdf', 'docx', 'rtf', 'jpg', 'jpeg', 'png', 'txt'];
+function readURL(input) {
+    if (input.files && input.files[0]) {
+        var extension = input.files[0].name.split('.').pop().toLowerCase(),
+            isSuccess = fileTypes.indexOf(extension) > -1;
+
+        if (isSuccess) {
+            var reader = new FileReader();
+            reader.onload = function (e) {
+                if (extension == 'pdf'){
+                    $(input).closest('.fileUpload').find(".icon").attr('src','images/web/file_pdf.svg');
+                }
+                else if (extension == 'docx'){
+                    $(input).closest('.fileUpload').find(".icon").attr('src','images/web/file_doc.svg');
```

Comment:
```suggestion
                } else if (extension == 'docx'){
                    $(input).closest('.fileUpload').find(".icon").attr('src','images/web/file_doc.svg');
```
Mấy cái else if nữa nè :P